vioscsi/vioscsi.c

```diff
@@ -485,7 +485,16 @@ ENTER_FN();
 
     /* Set num_queues and seg_max to some sane values, to keep "Static Driver Verification" happy */
     adaptExt->scsi_config.num_queues = 1;
-    adaptExt->scsi_config.seg_max = MAX_PHYS_SEGMENTS + 1;
+    /* Happy? (Begin) */
+    /* Getting virtqueue_size is prioritized over your happiness. I don't care. */
+    adaptExt->max_physical_breaks = min(
+                                        max(SCSI_MINIMUM_PHYSICAL_BREAKS, adaptExt->scsi_config.seg_max),
+    /* It is unknown that whether clamping to MAX_PHYS_SEGMENTS (+1) is necessary. For now stick to the habit. */
+                                        // SCSI_MAXIMUM_PHYSICAL_BREAKS);
```

Comment:
We can consider `128` as well:
https://github.com/torvalds/linux/blob/v5.8/include/linux/scatterlist.h#L329
https://github.com/torvalds/linux/blob/v5.8/include/scsi/scsi_host.h#L383 (`128 * 4096 / 512 = 1024`)